commit 212f6c8206bd4dcc7d8970c5f1632823ae4e783a
Author: Michael Forney <mforney@mforney.org>
Date:   Sun Feb 12 23:01:21 2017 -0800

Fix build with HAVE_GL==0

video/out/opengl/hwdec.h includes video/out/opengl/common.h, which tries
to include opengl headers.

diff --git a/options/options.c b/options/options.c
index b711156c1..0bb36ad3f 100644
--- a/options/options.c
+++ b/options/options.c
@@ -40,7 +40,6 @@
 #include "stream/stream.h"
 #include "video/csputils.h"
 #include "video/hwdec.h"
-#include "video/out/opengl/hwdec.h"
 #include "video/image_writer.h"
 #include "sub/osd.h"
 #include "audio/filter/af.h"
@@ -53,6 +52,10 @@
 #include "video/out/drm_common.h"
 #endif

+#if HAVE_GL
+#include "video/out/opengl/hwdec.h"
+#endif
+
 extern const char mp_help_text[];

 static void print_version(struct mp_log *log)
